### DISCUSSION TITLE
Javascript Select not working when conditionally disabled - Possible bug?

### DISCUSSION THREAD

**Discussion by diogogpinto** [filamentphp/filament#8590] [Help]
After updating Filament to v3.0.53, the javascript select stopped working as expected.

When I conditionally disable a Select (if it has no options) based on other select and use the native(false) method, the dependent select doesn't update.

Here's how to reproduce this behaviour:

Create a form and add the following schema:

```php
Forms\Components\Select::make('select_a')
    ->options(['Enable Second Select' => 'show', 'Disable Second Select' => 'hide'])
    ->live()
    ->native(false),
Forms\Components\Select::make('select_b')
    ->options(fn (Forms\Get $get): array => match ($get('select_a')) {
        'show' => ['option 1', 'option 2', 'option 3'],
        default => [],
    })
    ->disabled(fn (Forms\Components\Component $component) => empty($component->getOptions()))
    ->required(fn (Forms\Components\Component $component) => empty(!$component->getOptions()))
    ->native(false),
```

It should enable the select when select_a is working, but it isn't. In previous version of filament it was working.

Maybe I've done something wrong when upgrading.

**Comment by leandrocfe** [comment]
Can you create an [issue](https://github.com/filamentphp/filament/issues) please?

**Reply by diogogpinto** [reply]
@leandrocfe I will eventually, I couldn't because I don't have a repo available to create the issue. Will get back to it when away from the office.

**Comment by maytham553-elite** [comment]
do you solve it ?
